frontend/src/v4/routes/viewerGui/components/sequences/sequences.component.tsx

```diff
@@ -60,21 +61,44 @@ interface IProps {
 	draggablePanels: string[];
 	toggleLegend: () => void;
 	resetLegendPanel: () => void;
+	openOnToday: boolean;
 }
 
 const da =  new Date();
 
 const SequenceDetails = ({
 	startDate, endDate, selectedDate, selectedEndingDate, setSelectedDate, stepInterval, stepScale, setStepInterval,
-	setStepScale, currentTasks, loadingFrameState, loadingViewpoint, rightPanels, toggleActivitiesPanel,
+	setStepScale, currentTasks, loadingFrameState, loadingViewpoint, rightPanels, toggleActivitiesPanel, openOnToday,
 	fetchActivityDetails, onPlayStarted, frames, isActivitiesPending, toggleLegend, draggablePanels
-}) => (
+}) => {
+	const [dateToUse, setDateToUse] = useState(null);
+
+	useEffect(() => {
+		if (!openOnToday) {
+			setDateToUse(selectedDate);
+			return;
+		}
+
+		const now = new Date();
+		const newDateToUse = getDateWithinBoundaries(now, new Date(startDate), new Date(endDate));
+		setDateToUse(newDateToUse);
+
+		if (newDateToUse.getTime() !== now.getTime()) {
+			DialogsActionsDispatchers.open('info', MODAL_TODAY_NOT_AVAILABLE_BODY);
```

Comment:
I entered a sequence and I got this :
<img width="794" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/55a82cd9-4762-4d0c-b4c9-5b7b06618faf">